### PR TITLE
feat: new enclave UI

### PR DESCRIPTION
**Pull request by Strnadj**

### CODE REVIEW

File: apps/idos-enclave/src/pages/methods/Passkey.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      <div className="flex flex-col space-y-4">
+        {flow === "new" && (
+          <>
+            <Heading>Create your idOS key</Heading>
+
+            <Paragraph>Please click the button below to create an idOS passkey.</Paragraph>
+
+            <Button onClick={this.getPasskey}>Create passkey</Button>
+
+            <Collapsible title="What is a passkey?" Icon={QuestionMarkCircleIcon}>
```
Comment (juliosantos) [inline apps/idos-enclave/src/pages/methods/Passkey.tsx:148]: Please make this smaller — it almost looks more important than the button :)

**Review by juliosantos — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -32,6 +32,9 @@ yarn-error.log*
 # vercel
 .vercel
 
+# vite timestamped configs
+vite.config.ts.timestamp-*
```
[the remark this answers is not in the document]
Comment (Strnadj) [reply .gitignore:36]: @ditoglez do you know why this file has been created? ...

File: apps/idos-example-dapp/src/main.js [hunk 3]
```diff
@@ -89,7 +89,11 @@ const connectWallet = {
    *
    */
   const idos = await idOS.init({
-    container: "#idos-container",
+    enclaveOptions: {
+      container: "#idos-container",
+      theme: "dark",
+      mode: "new",
```
Comment (juliosantos) [inline apps/idos-example-dapp/src/main.js:95]: whoops

File: packages/idos-sdk-js/.env [hunk 4]
```diff
@@ -1,4 +1,5 @@
-VITE_IDOS_ENCLAVE_URL=https://enclave.idos.network
+# VITE_IDOS_ENCLAVE_URL=https://enclave.idos.network
+VITE_IDOS_ENCLAVE_URL=https://localhost:5174/
```
Comment (juliosantos) [inline packages/idos-sdk-js/.env:2]: whoops

**Review by juliosantos — CHANGES_REQUESTED** [review]

**Review by juliosantos — APPROVED** [review]

**Pull request merged**